packages/flame_3d/lib/src/graphics/graphics_device.dart

```diff
@@ -50,7 +50,7 @@ class GraphicsDevice {
 
   /// Must be set by the rendering pipeline before elements are bound.
   /// Can be accessed by elements in their bind method.
-  Iterable<Light> lights = [];
+  final LightingInfo lightingInfo = LightingInfo();
```

Comment:
```suggestion
  final lightingInfo = LightingInfo();
```

Reply:
this is actually in line with the style no?

![image](https://github.com/user-attachments/assets/7719c964-a669-4c98-b9fb-20abf6a54010)

Reply:
resolving as that is in line with how it is used elsewhere